### PR TITLE
@mzikherman => Few more article things

### PR DESCRIPTION
**Pull request by mzikherman**

### CODE REVIEW

**Comment by mzikherman** [comment]
Paired with @sweir27 !

File: schema/article.js [hunk 1]
```diff
@@ -29,6 +29,13 @@ let ArticleType = new GraphQLObjectType({
     thumbnail_image: {
       type: Image.type,
       resolve: ({ thumbnail_image }) => thumbnail_image
+    },
+    slug: {
+      type: GraphQLString
+    },
```
Comment (dzucconi) [inline schema/article.js]: This is called slug, and not ID?

Comment (mzikherman) [inline schema/article.js]: Yes, it's called slug

Comment (mzikherman) [inline schema/article.js]: I guess for consistency maybe I could have called this `id` here, and just returned `slug`. Consistent w/ Gravity, which is the bulk of the endpoints.

Probably all schemas should have `cached`, `id` and `_id` and we can call it a convention.

**Pull request merged**